config/samples/ibm-mq/ibmmq-server.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
```

Comment:
can we not make an ibm-mq folder in the samples? and just have it under `Sources` and `Targets` ? like all the rest of them?